packages/components/checkbox/src/checkbox-group.vue

```diff
@@ -53,6 +64,19 @@ const modelValue = computed({
   },
 })
 
+const aliasProps = computed(() => ({
+  ...checkboxDefaultProps,
+  ...props.props,
+}))
+const getOptionProps = (option: Record<string, any>) => {
+  const base = {
+    label: option[aliasProps.value.label],
+    value: option[aliasProps.value.value],
+    disabled: option[aliasProps.value.disabled],
+  }
+  return { ...option, ...base }
```

Comment:
We can now optimize this:
```suggestion
  return {
    label: option[aliasProps.value.label],
    value: option[aliasProps.value.value],
    disabled: option[aliasProps.value.disabled],
  }
```

Reply:
Other options should be included.

Reply:
> Thanks, didn't notice it.
> 
> Then we should update the documentation, currently it mention only label, value and disabled.
> 
> We can refer to cascader to mention the props (or partial props allowed):
> 
> <img alt="Image" width="1241" height="182" src="https://private-user-images.githubusercontent.com/91417411/483733143-66907c76-5d8b-4b4e-b120-ef9100c6ee01.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY0OTE4NTIsIm5iZiI6MTc1NjQ5MTU1MiwicGF0aCI6Ii85MTQxNzQxMS80ODM3MzMxNDMtNjY5MDdjNzYtNWQ4Yi00YjRlLWIxMjAtZWY5MTAwYzZlZTAxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA4MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwODI5VDE4MTkxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc3NTVmM2EwMjIyYjRkNGYzYjgxZDNkY2ZjMDM1YjQwYTNiNjI2YTBjZTNlYWJlN2M2MjRmMDBjMTNiZmZkOWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.SYPqFxEH91bIdmZYWFw8_K9__W24sl6AhIODiyiPOkY">

Thank you for your feedback 🙏. Personally, I think it’s sufficient to just support more configuration options through options here. As for prop alias configuration, keeping the commonly used three — label, value, and disabled — should already cover most needs without making things too bloated. What do you think?